ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/workers-rhel8/deprovision/cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8-deprovision-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-workers-rhel8-deprovision
+  steps:
+  - ref: gather-installer-boostrap-logs-in-bastion
+  - chain: cucushift-installer-rehearse-ibmcloud-ipi-deprovision
```

Comment:
@MayXuQQ could you help to confirm the RHEL machines also being removed along with the  `ibmcloud-ipi-deprovision` chain?

Reply:
[.openshift_install.log](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/39088/rehearse-39088-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-ibmcloud/1656310684377419776/artifacts/installer-rehearse-ibmcloud/ipi-deprovision-deprovision/artifacts/.openshift_install.log) 

> time="2023-05-10T16:20:08Z" level=info msg="Deleted instance \"ci-op-k4mr83q6-57523-ll7tq-rhel-2\""
time="2023-05-10T16:20:08Z" level=info msg="Deleted instance \"ci-op-k4mr83q6-57523-ll7tq-rhel-1\""
time="2023-05-10T16:20:08Z" level=info msg="Deleted instance \"ci-op-k4mr83q6-57523-ll7tq-master-0\""
time="2023-05-10T16:20:08Z" level=info msg="Deleted instance \"ci-op-k4mr83q6-57523-ll7tq-master-1\""
time="2023-05-10T16:20:08Z" level=info msg="Deleted instance \"ci-op-k4mr83q6-57523-ll7tq-master-2\""
time="2023-05-10T16:20:08Z" level=info msg="Deleted instance \"ci-op-k4mr83q6-57523-ll7tq-rhel-3\""